src/ontology/mondo-edit.obo

```diff
@@ -6931,19 +6931,16 @@ is_a: MONDO:0012000 {source="DOID:0060048", source="EFO:1001903"} ! specific pho
 
 [Term]
 id: MONDO:0000601
-name: autoimmune disorder of urogenital tract
-def: "A hypersensitivity reaction type II disease that involves the genitourinary system." [MONDO:patterns/location]
+name: obsolete autoimmune disorder of urogenital tract
+def: "OBSOLETE. A hypersensitivity reaction type II disease that involves the genitourinary system." [MONDO:patterns/location]
 comment: Reason: grouping class. Term to consider: none
-subset: obsoletion_candidate
 synonym: "autoimmune disease of genitourinary system" EXACT [MONDO:design_pattern]
 synonym: "genitourinary system autoimmune disease" EXACT [MONDO:design_pattern, MONDO:patterns/location]
 synonym: "genitourinary system hypersensitivity reaction type II disease" EXACT [MONDO:patterns/location]
-xref: DOID:0060049 {source="MONDO:equivalentTo"}
-is_a: MONDO:0007179 {source="DOID:0060049", source="MONDO:Entailed", source="MONDO:Redundant"} ! autoimmune disease
-intersection_of: MONDO:0007179 ! autoimmune disease
-intersection_of: disease_has_location UBERON:0004122 ! genitourinary system
+xref: DOID:0060049 {source="MONDO:obsoleteEquivalent"}
 relationship: seeAlso https://github.com/monarch-initiative/mondo/issues/5024
-property_value: IAO:0006012 "2022-09-01" xsd:string
+property_value: IAO:0000231 "out of scope" xsd:string
```

Comment:
```suggestion
property_value: IAO:0000231 "out of scope" xsd:string {source="MONDO:excludeGroupingClass"}
```
I added the exclusion reason.

@sabrinatoro I just learned how to do these inline commits, I think it is so cool! Nico told me about it and I added it to the pull request tutorial: https://oboacademy.github.io/obook/tutorial/pull-requests/#inline-commits

Reply:
Hm, it is not letting me commit though...
@sabrinatoro Can you try to commit it?